firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/PersonalizationTest.java

```diff
@@ -99,12 +105,16 @@ public void logArmActive_singlePersonalizationKey_loggedOnce() {
     verify(mockAnalyticsConnector, times(1))
         .logEvent(
             eq(ANALYTICS_ORIGIN_PERSONALIZATION), eq(ANALYTICS_PULL_EVENT), any(Bundle.class));
-    assertThat(FAKE_LOGS).hasSize(1);
+    verify(mockAnalyticsConnector, times(1))
```

Comment:
Do we want to verify the other (internal) log event here, too?

Reply:
👍  I'd vote to use the `_loggedInternallyAndExternally` suffix for the test below too, ie `logArmActive_multiplePersonalizationKeys_loggedMultiple` --> `logArmActive_multiplePersonalizationKeys_loggedInternallyAndExternally`